pkgs/development/tools/sshs/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "sshs";
+  version = "3.2.0";
+
+  src = fetchFromGitHub {
+    owner = "quantumsheep";
+    repo = pname;
+    rev = version;
+    sha256 = "D9doNVb2sTnzM8tF8cSJbIoaIYjGurkUHEyhcE3OqQg=";
+  };
+
+  vendorSha256 = "QWFz85bOrTnPGum5atccB5hKeATlZvDAt32by+DO/Fo=";
+
+  meta = with lib; {
```

This will allow for updating via [nixpkgs-update](https://github.com/ryantm/nixpkgs-update).

```suggestion
  passthru.updateScript = nix-update-script {
    attrPath = pname;
  };

  meta = with lib; {
```

---

resolved in 3fae311